### DISCUSSION TITLE
Multiple synth-fun sharing the same grammar

### DISCUSSION THREAD

**Discussion by zhengyao-lin** [cvc5/cvc5#10811] [Q&A] [answered]
Hello! This is more of a SyGuS format question. My application requires sending multiple `synth-fun` commands with the same (and somewhat complicated) grammar. Is there a short hand for these `synth-fun` to share the same grammar?

Thank you!

**Comment by ajreynol** [comment, marked as answer]
No, this feature isn't available at the moment.  We have considered adding a command
```
(define-grammar <symbol> <sorted-var-list> <sorted-var-list> <grouped-rule-listing>)
```
where this would bind <symbol> to the grammar, making it able to be referenced later.

We will keep you posted when this becomes available.
